src/test/java/com/madgag/gif/fmsware/TestAnimatedGifEncoder.java

```diff
@@ -83,6 +114,22 @@ private void encodeSampleSonicFrames() {
         encoder.finish();
     }
 
+    private void encodeSampleExactFrames() {
+        encoder.setRepeat(0);
+        encoder.setDelay(1000);
+        encoder.addFrame(agif);
+        encoder.addFrame(bgif);
+        encoder.finish();
+    }
+
+    private void encodeSampleExactFramesTransparent() {
```

Comment:
`encodeSampleExactFrames()` seems identical to `encodeSampleExactFramesTransparent()` - we should avoid code duplication?